desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/Breadcrumb/Breadcrumb.tsx

```diff
@@ -26,9 +28,18 @@ interface BreadcrumbProps {
 }
 
 const Breadcrumb: React.FC<BreadcrumbProps> = ({ label, url, onFilepathChange }) => {
+  const onClick = () => {
```

Comment:
Nitty gritty: So when you are handling an event, i.e. implementing the callback in a function, then we shouldn't call it onXXX() but rather handleFilepathChange or something, that in turns calls the onFilepathChange(url);

Think of it like this, the child component calls a click callback when it is clicked, that is all it has to know. It does not know what other components will do with that callback. The parent component on the other hand, it knows that this click means that we should attach a url or a function called onFilepathChange, so we choose a more specific name for this function.